### PR TITLE
Add migration to switch to uuid tokens during upgrade

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

**Comment by jsuchome** [comment]
LGTM

**Comment by toabctl** [comment]
+1

File: chef/data_bags/crowbar/migrate/keystone/035_pki_uuid_switch.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+def upgrade(ta, td, a, d)
+  # This migration (if part of the release) will only run when upgrading
+  # from Tex. Switch keystone to use UUID tokens after the upgrade for those
+  # systems
```
Comment (vuntz) [inline chef/data_bags/crowbar/migrate/keystone/035_pki_uuid_switch.rb]: Can you add a mention of CVE-2015-7546 to this comment?

**Comment by jsuchome** [comment]
Tested the upgrade with keystone, and the migration worked correctly. +1

**Comment by aplanas** [comment]
+1

**Comment by dguitarbite** [comment]
+1 works for me.

File: chef/data_bags/crowbar/migrate/keystone/035_pki_uuid_switch.rb (hunk 1 above)
Comment (vuntz) [inline chef/data_bags/crowbar/migrate/keystone/035_pki_uuid_switch.rb]: https://github.com/crowbar/crowbar-openstack/pull/268

**Pull request merged**